api/views/share.py

```diff
@@ -127,3 +130,11 @@ def get(self, request, *args, **kwargs):
                 return HttpSmartResponsePermanentRedirect(url)
             return HttpSmartResponseRedirect(url)
         return http.HttpResponseGone()
+
+
+class ModelTypesView(views.APIView):
+    def get(self, request, *args, **kwargs):
+        yaml_file = os.path.abspath('../SHARE/share/models/creative.yaml')
+        with open(yaml_file) as fobj:
+            model_specs = yaml.load(fobj)
```

Comment:
This should be moved into the class definition so that it will only load once, on start up.